ui/pages/Public/PrivacyPolicy.jsx

```diff
@@ -0,0 +1,402 @@
+import React from 'react'
+import { Header, Segment, List } from 'semantic-ui-react'
+
+export default () =>
+  <Segment basic padded="very">
```

Comment:
The text for this policy comes from nd is approved by Broad legal. Theres no real need to review it